tests/update_testapps_dependencies.go

```diff
@@ -0,0 +1,80 @@
+package main
```

Comment:
The runs only one command... I think using simple bash or ps scripts would be simpler here.

Reply:
I thought we will have to maintain two separate scripts for Linux and Windows and hence used a go file.

Reply:
I believe maintaining script is easier than maintaining go code. For example, if we need more updates in the other files, script is easier. also we recommend to install mingw for windows environment. there are bash executable in mingw. so having only one bash script will not have any issues in Windows.

Reply:
will move to .sh file